src/views/Biobanker.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <navigation-bar />
+    <div class="container">
+        <NegotiationList :negotiations="negotiations"/>
+    </div>
+</template>
+
+<script>
+import NegotiationList from "@/components/NegotiationList.vue";
+import NavigationBar from "@/components/NavigationBar.vue";
+import {mapActions, mapGetters} from "vuex";
+
+export default {
+    name: "Biobanker",
+    components: {
+        NavigationBar,
+        NegotiationList
+    },
+    methods: {
+        ...mapActions(["retrieveBiobankerRoleNegotiations"])
+    },
+    computed: {
+        ...mapGetters({
+            negotiations: "getNegotiations",
+        })
+    },
+    async mounted() {
+        await this.retrieveBiobankerRoleNegotiations();
+    },
+}
+</script>
+
+<style scoped>
+
+.container{
```

Comment:
better to use bootstrap's classes to define dimensions, margins, etc.

https://getbootstrap.com/docs/5.0/layout/containers/